plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/tools/internal/editor/tabbar/actions/TabbarArrangeMenuManager.java

```diff
@@ -164,13 +164,16 @@ private void refreshArrangeMenu(IWorkbenchPage page, ISelection selection) {
         }
 
         if (arrangeSelection) {
-            if (!isArrangeSelection()) {
+            if (!isArrangeSelectionAndBorderNodes()) {
```

Comment:
I do not understand the aim of this method

Reply:
To tell the truth, neither do I, but the change is consistent.

Reply:
I think this function checks whether actions have already been added.